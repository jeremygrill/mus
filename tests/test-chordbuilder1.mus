Int main(){
	
	Chord c;
	Note x;
	Note y;
	Int i;
	Int r;

	x = (11,11|11);
	y = (22,22|22);
	i = 0;

	while(i < 10){
		c = c * (10+i,10|10);
		c = (100-i,10|10) * c;
		i = i + 1;
	}

	printc(c);
	return 0;
}