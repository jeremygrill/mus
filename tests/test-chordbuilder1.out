[(10,10|10)(100,10|10)(11,10|10)(99,10|10)(12,10|10)(98,10|10)(13,10|10)(97,10|10)(14,10|10)(96,10|10)(15,10|10)(95,10|10)(16,10|10)(94,10|10)(17,10|10)(93,10|10)(18,10|10)(92,10|10)(19,10|10)(91,10|10)]
